testbed/tests/scenarios.go

```diff
@@ -158,9 +158,15 @@ func Scenario10kItemsPerSecond(
 	tc.StartAgent()
 
 	if loadOptions.DataItemsPerSecond == 0 {
-		// Use 10k spans or metric data points per second by default.
+		// Use 10k spans or metrics per second by default.
 		loadOptions.DataItemsPerSecond = 10000
 	}
+
+	if loadOptions.ItemsPerBatch == 0 {
+		// Use 100 spans or metrics per batch by default.
+		loadOptions.ItemsPerBatch = 100
```

Comment:
Instead of setting this here we can alter the default in LoadGenerator.Start (note that it will affect metrics as well). Do you only want this change in this particular scenario or globally change the default for all tests?

Reply:
Done. I hardcoded the values for this particular scenario Scenario10kItemsPerSecond and remove the ability to configure it for this scenario.